model/contract/Constraint.ttl

```diff
@@ -71,10 +77,24 @@ ids:unit rdfs:subPropertyOf odrl:unit;
 ids:pipEndpoint a owl:ObjectProperty;
      rdfs:label "has PIP endpoint"@en;
      rdfs:domain ids:Constraint;
-     rdfs:range xsd:anyURI ;
+     rdfs:range ids:PipEndpoint ;
      rdfs:comment "The reference to the endpoint which provides the current state of the feature of interest (as referrenced by the leftOperand) can be retrieved."@en
 .
 
+ids:pipInterfaceDescription a owl:ObjectProperty;
+     rdfs:label "pip interface description"@en;
+     rdfs:domain ids:PipEndpoint;
+     rdfs:range  xsd:anyURI;
+     rdfs:comment "The reference to a URI that prodived the interface description of a PIP."@en;
+.
+
+ids:pipAccessURI a owl:ObjectProperty;
```

Comment:
This property is so similar to `ids:pipAccessURI` that I would consider merging them to a single/common property `ids:accessURI`. If we really need to distinguish between the access URI for a PIP vs. a PXP, then we could still have two subproperties of `ids:accessURI`.

Reply:
@clange Which exact property are you referring to with "this property"?

Reply:
@clange @hosseinzadeha  can you please review the UsageControlEndpoint.ttl in Contract folder?